internal/handlers/common/filter.go

```diff
@@ -452,3 +471,20 @@ func filterFieldExprBitsAnySet(fieldValue, maskValue any) (bool, error) {
 
 	return true, nil
 }
+
+// filterFieldExprExists handles {field: {$exists: value}} filter.
+func filterFieldExprExists(fieldExist bool, exprValue any) (bool, error) {
+	expr, ok := exprValue.(bool)
+	if !ok {
+		return true, nil
```

Comment:
Without comment that looks like a typo

Reply:
That's just MongoDB behavior. 😄

Reply:
I understand that. But there should be a comment

Reply:
Sure, I'll add one.